src/quoridor/controller/gamecontroller.go

```diff
@@ -5,50 +5,93 @@ import (
 
 	"quoridor/game"
 	"quoridor/storage"
-
-	"github.com/lithammer/shortuuid"
 )
 
+type Party struct {
+	game game.Game
+	players map[string]int
+}
+
+func (p Party) isReady() bool {
+	return len(p.players) == len(p.game.Pawns)
+}
+
+func getParty(id string) (Party, error) {
+	p, found := storage.Get(id)
+	if !found {
+		return Party{}, errors.New("The game does not exist")
+	}
+	return p.(Party), nil
+}
+
+func checkPlayer(p Party, playerID string) error {
+	player := p.players[playerID]
+	if player == 0 {
+		return errors.New("Forbidden")
+	}
+	if !p.isReady() {
+		return errors.New("Game is not ready")
+	}
+	if player != p.game.PlayerTurn {
+		return errors.New("It is not your turn")
+	}
+	return nil
+}
+
 // CreateGame create a game with the default configuration
-func CreateGame(conf *game.Configuration) (*game.Game, error) {
-	boardSize := conf.BoardSize
-	board, err := game.NewBoard(boardSize)
+func CreateGame(conf game.Configuration, playerID string) (*game.Game, error) {
+	game, err := game.NewGame(conf)
 	if err != nil {
 		return nil, err
 	}
-	lineCenter := (boardSize - 1) / 2
-	pawn := game.Pawn{game.Position{0, lineCenter}}	
-	id:= shortuuid.New()
-	game := game.Game{id, false, pawn, []game.Fence{}, board}
-	storage.Set(game.ID, game)
+	players := make(map[string]int)
+	players[playerID] = 1
+	storage.Set(game.ID, Party{game, players})
 	return &game, nil
 }
 
 // GetGame get the game via its identifier
 func GetGame(id string) (game.Game, error) {
-	g, found := storage.Get(id)
-	if !found {
-		return game.Game{}, errors.New("The game does not exist")
+	p, err := getParty(id)
+	if err != nil {
+		return game.Game{}, err
+	}
+	return p.game, nil
+}
+
+func JoinGame(id string, playerID string) (game.Game, error) {
```

Comment:
Using "getParty" in "JoinGame" is a little bit confusing. You can maybe change one of this names?